Makefile

```diff
@@ -58,7 +58,7 @@ test-compile-changed: os-autoinst/
 
 .PHONY: test-yaml-valid
 test-yaml-valid:
-	$(eval YAMLS=$(shell sh -c "git ls-files schedule/ test_data/ | grep '\\.ya\?ml$$'"))
+	$(eval YAMLS=$(shell sh -c "git ls-files schedule/ test_data/ | grep -v '/defaults/' | grep '\\.ya\?ml$$'"))
```

Comment:
This is wrong in a first sight. It makes the CI obsolete and probe to give false positives

Reply:
The problem here is that the enhancement for the scheduler uses a new format for the default files which differs from the current schema used for YAML-Schedules. See [my documentation on Confluence](https://confluence.suse.com/display/QYT/How-To%3A+Migrating+test+suites+to+the+new+scheduler) for details. So "schedule:" is no longer an array of type string, but rather a hash with sections that are arrays of string. 
Therefore the YAML checking for the /defaults/ directory is omitted.

Reply:
but @b10n1k is right, in my initial PoC I just wrote that pass the CI, we will plan some ticket for it or I will improve it directly there, we need some specific path not so generic :) so no worries, was in my mind! CI/Documentation/Schema check that will come later, we are not going to merge anything with `todos` :)